patches/sequelize-test-helpers+1.3.3.patch

```diff
@@ -0,0 +1,314 @@
+
+ class Model {}
+-Model.init = spy()
+-Model.belongsToMany = spy()
+-Model.belongsTo = spy()
+-Model.hasMany = spy()
+-Model.hasOne = spy()
++Model.init = isJestRunner ? jest.fn() : spy()
++Model.belongsToMany = isJestRunner ? jest.fn() : spy()
++Model.belongsTo = isJestRunner ? jest.fn() : spy()
++Model.hasMany = isJestRunner ? jest.fn() : spy()
++Model.hasOne = isJestRunner ? jest.fn() : spy()
+
+ module.exports = { Model, DataTypes }
+diff --git a/node_modules/sequelize-test-helpers/src/sequelize.js b/node_modules/sequelize-test-helpers/src/sequelize.js
+index b3983c9..a654f77 100644
+--- a/node_modules/sequelize-test-helpers/src/sequelize.js
++++ b/node_modules/sequelize-test-helpers/src/sequelize.js
+@@ -1,11 +1,12 @@
+-const sinon = require('sinon')
+ const hooks = require('./constants/hooks')
+ const staticMethods = require('./constants/staticMethods')
+ const { syncMethods, asyncMethods } = require('./constants/staticModelMethods')
++const { isJestRunner, sinon } = require('./utils/checkIsJestRunner')
+
+ const sequelize = {
+   define: (modelName, modelDefn, metaData = {}) => {
+-    const model = function () {}
++    const model = function() {
++    }
+     model.modelName = modelName
+
+     const attachHook = name => hook => {
+@@ -19,7 +20,7 @@ const sequelize = {
+     }
+
+     const addStatic = key => {
+-      model[key] = sinon.stub()
++      model[key] = isJestRunner ? jest.fn() : sinon.stub()
+     }
+
+     hooks.forEach(hook => {
+@@ -34,11 +35,11 @@ const sequelize = {
+     syncMethods.forEach(addStatic)
+     asyncMethods.forEach(addStatic)
+
+-    model.isHierarchy = sinon.spy()
++    model.isHierarchy = isJestRunner ? jest.fn() : sinon.spy()
+
+-    model.prototype.update = sinon.stub()
+-    model.prototype.reload = sinon.stub()
+-    model.prototype.set = sinon.spy()
++    model.prototype.update = isJestRunner ? jest.fn() : sinon.stub()
++    model.prototype.reload = isJestRunner ? jest.fn() : sinon.stub()
++    model.prototype.set = isJestRunner ? jest.fn() : sinon.spy()
+     Object.keys(modelDefn).forEach(attachProp)
+
+     model.prototype.indexes = metaData.indexes
+@@ -49,7 +50,7 @@ const sequelize = {
+ }
+
+ staticMethods.forEach(method => {
+-  sequelize[method] = sinon.stub()
++  sequelize[method] = isJestRunner ? jest.fn() : sinon.stub()
+ })
+
+ module.exports = sequelize
+diff --git a/node_modules/sequelize-test-helpers/src/utils/checkIsJestRunner.js b/node_modules/sequelize-test-helpers/src/utils/checkIsJestRunner.js
+new file mode 100644
+index 0000000..eb32257
+--- /dev/null
++++ b/node_modules/sequelize-test-helpers/src/utils/checkIsJestRunner.js
+@@ -0,0 +1,7 @@
++const isJestRunner = process.env.SEQUELIZE_TEST_HELPERS_JEST_RUNNER === 'true' || process.env.SEQUELIZE_TEST_HELPERS_JEST_RUNNER === true
++
++module.exports = {
++  isJestRunner,
++  expect: isJestRunner ? expect : require('chai').expect,
++  sinon: isJestRunner ? () => {} : require('sinon')
++}
```

Comment:
Why is this file here after merging the entity-tests branch??? I saw a couple other inconsistencies as well. Is that possible to fix?

Reply:
The base branch wasn't merged in after approval the approval for #287. Fixed.